src/components/DemoComponent/DemoComponent.test-int.js

```diff
@@ -1,4 +1,4 @@
-/// <reference path="./test/codecept/steps.d.ts" />
+// / <reference path="./test/codecept/steps.d.ts" />
```

Comment:
linter is trying to fix this, but it's not a mistake according to Rizwan?  I think is a TypeScript directive. Wait a minute until we clarify